scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -104,9 +106,44 @@ class RedundantBraces(implicit ctx: RewriteCtx) extends RewriteSession {
       case t: Term.Interpolate if settings.stringInterpolation =>
         processInterpolation(t)
 
+      case i: Importer =>
+        processImport(i)
+
       case _ =>
     }
 
+  private def processImport(tree: Importer) = {
+
+    if (tree.importees.size == 1) {
+      tree.importees.head match {
+        case _: Importee.Rename | _: Importee.Unimport =>
+          val tokens = tree.importees.head.tokens
+
+          val start = ctx.tokenTraverser.findBefore(tokens.head) { t =>
+            if (t.is[LeftBrace]) Some(true)
+            else if (t.is[Dot]) Some(false)
+            else None
+          }
+
+          val end = ctx.tokenTraverser.findAfter(tokens.last) { t =>
+            if (t.is[RightBrace]) Some(true)
+            else if (t.is[LF]) Some(false)
+            else None
+          }
+
+          (start, end) match {
```

Comment:
to avoid doing both when one fails... would this be more readable:
```
for {
  lbrace <- ....findBefore {...}
  rbrace <- ...findAfter {...}
} yield {
  implicit val builder = Seq.newBuilder[TokenPatch]
  removeBraces(lbrace, rbrace)
  ctx.addPatchSet(builder.result(): _*)
}
```

alternatively:
```
  ...findBefore {
  }.foreach { lbrace =>
    ...findAfter {
    }.foreach { rbrace =>
       // remove braces
    }
  }
```

finally, should we simply look for preceding `LeftBrace` and then use `ctx.getMatching`?

Reply:
Decided to do a single foreach and then `ctx.getMatching`, looks better.